tests/ext/pcntl/pcntl_fork_long_running_autoflush.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+Long running autoflush
+--ENV--
+DD_TRACE_GENERATE_ROOT_SPAN=false
+DD_TRACE_AUTO_FLUSH_ENABLED=true
+--FILE--
+<?php
+
+require 'functions.inc';
```

Comment:
Do you think we could include the [fake tracer includes](https://github.com/DataDog/dd-trace-php/blob/8ac6800/tests/ext/sandbox/auto_flush.phpt#L13-L14) here to show the auto-flush working? I'm not sure if that would work as expected for the child process though.

Reply:
I went a little bit further and moved the various `fake_*` to the `includes` root folder so I could reuse them. I also added a fake span and scope. Let me know if you like the idea or we can follow a different approach. Addressed in https://github.com/DataDog/dd-trace-php/pull/1196/commits/68f0de355508fc1470188bc98dc9405a64210d7c and in the followup https://github.com/DataDog/dd-trace-php/pull/1196/commits/21bf71ab76df3ba6b24805e8ad4e2c37fdc791be